src/locales/en/account.json

```
@@ -10,6 +10,7 @@
   "CROP_IMAGE_MODAL_UNEXPECTED_ERROR": "An unexpected error occurred when trying to crop the image, please try again later",
   "MAIN_MENU": {
     "HOME": "Home",
+    "STATS": "Statistics",
```

Maybe we want a clearer key?

---

Do you mean change to
```suggestion
    "STATISTICS": "Statistics",
```

or 
```suggestion
    "STATISTICS_MAIN_MENU": "Statistics",
```
but it is already contextual no ? So I think short key is ok. I can use `STATISTICS` if you prefer.